app/client/src/common/utils.js

```diff
@@ -6,4 +6,8 @@ function titleCase(str: string): string {
   return str[0].toUpperCase() + str.slice(1)
 }
 
-export { titleCase }
+function isLengthGreaterThanOne(obj: any): boolean {
+  return obj.length > 1
+}
+
+export { titleCase , isLengthGreaterThanOne }
```

Comment:
I think this function is an unnecessary level of abstraction. I think just checking the array length right where its needed would be simpler (see below).

Reply:
Removed